src/plugins/recurring-issues/index.js

```diff
@@ -21,6 +21,9 @@ The next scheduled release will occur on ${releaseDate.format("dddd, MMMM Do, YY
 
 - [ ] Remove the 'tsc agenda' label on this issue
 - [ ] Review open pull requests and merge any that are [ready](https://eslint.org/docs/maintainer-guide/pullrequests#when-to-merge-a-pull-request)
+- [ ] Release \`@eslint/js\` with to match the upcoming \`eslint\` version in [Jenkins](https://jenkins.eslint.org)
+- [ ] Verify if there are other packages (i.e., \`espree\`, \`eslint-scope\`, \`eslint-visitor-keys\`) that need to be released first
```

Comment:
```suggestion
- [ ] Verify if there are other packages (i.e., \`espree\`, \`eslint-scope\`, \`eslint-visitor-keys\`, \`@eslint/eslintrc\`) that need to be released first
```

This one is also a dependency.